client/src/pages/ToursToday.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
+import { useHistory } from "react-router-dom";
+
+import styled from "styled-components/macro";
+import GlobalStyle from "../GlobalStyles";
+
+import { getCurrentDateString } from "../utils/date";
+import { getDataByQuery } from "../utils/api";
+
+import Badge from "../components/Badge";
+import Card from "../components/Card";
+import HeaderMain from "../components/HeaderMain";
+import ButtonPlus from "../components/ButtonPlus";
+
+const PageWrapper = styled.div`
+  position: fixed;
+  overflow: auto;
+  height: 100%;
+  width: 100%;
+  background: var(--gradient-dark);
+  & > *:not(:first-child) {
+    margin: 1rem auto;
+  }
+  & > :nth-child(2) {
+    margin-top: clamp(260px, 40vw, 400px);
+  }
+`;
+
+const ToursToday = () => {
+  const [Tours, setTours] = useState([]);
+  const history = useHistory();
+  useEffect(() => {
+    const doFetch = async () => {
+      const today = getCurrentDateString();
+      const todaysTours = await getDataByQuery({
+        collectionName: "tasks",
+        dataName: "date",
+        query: today,
+      });
+      setTours(todaysTours);
+    };
+    doFetch();
+  }, []);
+  return (
+    <>
+      <GlobalStyle />
+      <PageWrapper>
+        <HeaderMain />
+        {Tours &&
+          Tours.map((ride) => {
+            return (
+              <Card
+                key={ride._id}
+                type={ride.priority}
+                start={ride.start}
+                dest={ride.dest}
+                rider={ride.assignment}
```

Comment:
Why not always name it `rider` or `assignment`, or is there a different?

Reply:
good point but I think its useful to have a distinction between a Rider on a Card and a assignment for a Tour.